File: openapi/code_samples/javascript/api_v1_organizations_{organization_id}_clients/post.js
const response = await scalekit.m2m.createOrganizationClient('org_123', {
  name: 'GitHub Actions Deployment Service',
  description: 'Service account for GitHub Actions to deploy applications to production',
  scopes: ['deploy:applications', 'read:deployments'],
  audience: ['deployment-api.acmecorp.com'],
})

File: openapi/code_samples/go/api_v1_organizations_{organization_id}_clients/post.go
response, err := scalekitClient.M2M().CreateOrganizationClient(
  ctx,
  "org_123",
  scalekit.CreateOrganizationClientOptions{
    Name:        "GitHub Actions Deployment Service",
    Description: "Service account for GitHub Actions to deploy applications to production",
    Scopes:      []string{"deploy:applications", "read:deployments"},
    Audience:    []string{"deployment-api.acmecorp.com"},
  },
)
if err != nil {
  // handle error
  return
}
_ = response
